docs/how-to/installation/aws.txt

```diff
@@ -359,6 +359,8 @@ Master
 -  Allow EC2 to assume role
 -  Allow EC2 to dynamically create and terminate instances with agent
```

Comment:
blocking: can you also help update this line to be "Allow EC2 to describe, create and terminate instances with agent"

Reply:
Done